### ISSUE TITLE
Page section missing lang attribute

### ISSUE THREAD

**Issue by adorayi** [cds-snc/design-gc-conception#2116]
The Government of Canada logo on the top of every page, has a title that is in two languages (French and English): "Government of Canada / Gouvernement du Canada".

On the English language version of the page however, the French text does not have a French language attribute.

**Impact on Users:**

Screen readers use language attributes to know how to pronounce text correctly. If a webpage doesn’t specify the language of the content, the screen reader will guess based on the user’s default language. This can lead to mispronunciations or confusion if the actual text is in a different language.

**Sample Code** (required):
<title id="signature-en-title">Government of Canada / Gouvernement du Canada</title>

**Recommendation (required if not SAMS/OCMC):**
We recommend ensuring that in all cases where text strings involve languages other than the page language, language attributes are used to inform screen readers of the language the text strings are in.

**Additional Resources:**
https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html

**Steps to Reproduce:**
1. Open the solution: https://design-system.alpha.canada.ca/en/components/
2. Inspect the Government of Canada logo on the top of the page.
